cmd/zpool/zpool.d/iostat

```diff
@@ -15,24 +15,32 @@ if [ "$1" = "-h" ] ; then
 	exit
 fi
 
+if [ "$(uname)" = "FreeBSD" ]; then
+	k="-K"
+	y=""
+else
+	k="-k"
+	y="-y"
+fi
+
 if [ "$script" = "iostat-1s" ] ; then
 	# Do a single one-second sample
 	extra="1 1"
 	# Don't show summary stats
-	y="-y"
+	brief="$y"
 elif [ "$script" = "iostat-10s" ] ; then
 	# Do a single ten-second sample
 	extra="10 1"
 	# Don't show summary stats
-	y="-y"
+	brief="$y"
 fi
 
 if [ -f "$VDEV_UPATH" ] ; then
 	# We're a file-based vdev, iostat doesn't work on us.  Do nothing.
 	exit
 fi
 
-out=$(eval "iostat $y -k -x $VDEV_UPATH $extra")
+out=$(eval "iostat -d $k $brief -x $VDEV_UPATH $extra" | tail -n 2)
```

Comment:
I don't believe this will work.  In my version of `iostat` there appears to be an extra trailing new line, so `| tail -n 2` will not include the headers.  The behavior of `iostat` is different enough between the platform why don't we do this with an

```
if [ "$(uname)" = "Linux" ]; then
else
fi
```